AmazonChimeSDKDemo/AmazonChimeSDKDemo/data/TranscriptionStreamParams.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  TranscriptionStreamParams.swift
+//  AmazonChimeSDKDemo
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+public struct TranscriptionStreamParams: Codable {
+    let contentIdentificationType: String?
+    let contentRedactionType: String?
+    let enablePartialResultsStabilization: Bool?
+    let partialResultsStability: String?
+    let piiEntityTypes: String?
+    let languageModelName: String?
+    let identifyLanguage: Bool?
+    let languageOptions: String?
+    let preferredLanguage: String?
+
+    init(contentIdentificationType: String?, contentRedactionType: String?, enablePartialResultsStabilization: Bool?, partialResultsStability: String?,
+         piiEntityTypes: String?, languageModelName: String?, identifyLanguage: Bool?, languageOptions: String?, preferredLanguage: String?) {
+            
+        self.contentIdentificationType = contentIdentificationType
+        self.contentRedactionType = contentRedactionType
+        self.enablePartialResultsStabilization = enablePartialResultsStabilization
+        self.partialResultsStability = partialResultsStability
+        self.piiEntityTypes = piiEntityTypes
+        self.languageModelName = languageModelName
+        self.identifyLanguage = identifyLanguage
```

Comment:
should it be `identifiedLanguage`?

Reply:
no, this the parameter for the request: https://docs.aws.amazon.com/transcribe/latest/dg/API_streaming_StartStreamTranscription.html#:~:text=another%20partial%20result.-,IdentifyLanguage,-Optional.%20Set%20this